src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String colour;
+
+    public Ball(int number, String colour) {
+        this.number = number;
+        this.colour = colour;
+    }
+
+    public String toString() {
+        return "Ball number: " + this.number + " Ball colour: " + this.colour;

```

Comment:
```suggestion
        return "Ball number: " + number + " Ball colour: " + colour;
```

Why do we need `this` keyword here?